oauth/origin/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Redirecting to Google</title>
```

Comment:
This will need to be more generic when we support other oauth providers besides just google. I know this as created to solve a google specific problem, but I think theres definitely a world where this provides customers with a realllly easy solution to use oauth with Turnkey